spec/presenters/coronavirus/sub_section_json_presenter_spec.rb

```diff
@@ -86,6 +86,41 @@
     end
   end
 
+  describe "#build_link" do
+    context "given a normal link" do
+      it "builds a link hash for the publishing api" do
+        expected_output = { label: "title", url: "/link" }
+        expect(subject.build_link("title", "/link")).to eq(expected_output)
+      end
+    end
+
+    context "given a featured link" do
+      it "builds a link hash for the publishing api with a featured link" do
+        allow(subject).to receive(:description_for_featured_link).and_return("description")
+
+        sub_section.featured_link = "/link"
+        expected_output = { label: "title", url: "/link", description: "description", featured_link: true }
+        expect(subject.build_link("title", "/link")).to eq(expected_output)
+      end
+    end
+
+    context "when link is to a subtopic path" do
+      let(:business) { create :coronavirus_page, :business }
+      let(:fixture_path) { Rails.root.join "spec/fixtures/coronavirus_landing_page.yml" }
+      let(:description) { "Find out about the government response to coronavirus (COVID-19) and what you need to do." }
+      let(:path) { business.base_path }
```

Comment:
Minor: I don't think this value is needed as it's easy enough for the expectation to call `business.base_path`.

Reply:
sorted :)